src/main/java/com/shilymily/bms/controller/CompanyController.java

```diff
@@ -0,0 +1,46 @@
+package com.shilymily.bms.controller;
+
+import com.shilymily.bms.entity.Company;
+import com.shilymily.bms.service.CompanyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.ConstraintViolationException;
+import java.util.List;
+
+/**
+ * Created by hmaskai on 9/8/18.
+ */
+@RestController
+@RequestMapping("/master/company")
+public class CompanyController {
+
+    @Autowired
+    private CompanyService companyService;
+
+    @PostMapping(path = "/add", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Let's create constants for all the paths (endpoints) and use them

Reply:
Overkill. Will add swagger integration for making sure all the endpoints are exposed